beacon-chain/operations/slashings/service.go

```diff
@@ -194,11 +196,7 @@ func (p *Pool) validatorSlashingPreconditionCheck(
 		return false, nil
 	}
 	// Checking if the validator has already been slashed.
-	slashedValidators := state.Slashings()
```

Comment:
This is a bug - we should not be checking the slashings list, instead just check if the validator was slashed